edgelet/build/windows/build-diagnostics.ps1

```diff
@@ -19,8 +21,8 @@ $env:NO_VALGRIND = 'true'
 
 $originalRustflags = $env:RUSTFLAGS
 $env:RUSTFLAGS += ' -C target-feature=+crt-static'
-Write-Host "$cargo build -p iotedge-diagnostics --release --manifest-path $ManifestPath"
-Invoke-Expression "$cargo build -p iotedge-diagnostics --release --manifest-path $ManifestPath"
+Write-Host "$cargo build -p iotedge-diagnostics --$BuildConfiguration --manifest-path $ManifestPath"
```

Comment:
I don't think `--debug` is a valid argument to `cargo build`. You either include or omit `--release`.